metricflow/dataflow/builder/dataflow_plan_builder.py

```diff
@@ -225,13 +228,42 @@ def _build_derived_metric_output_node(
             f"For {metric.type} metric: {metric_spec}, needed metrics are:\n"
             f"{pformat_big_objects(metric_input_specs=metric_input_specs)}"
         )
+
+        parent_nodes: List[BaseOutput] = []
+
+        for metric_input_spec in metric_input_specs:
+            # TODO: See: https://github.com/dbt-labs/metricflow/issues/881
+            if (metric_spec.offset_to_grain is not None or metric_spec.offset_to_grain is not None) and (
+                metric_input_spec.offset_window is not None or metric_input_spec.offset_to_grain is not None
```

Comment:
This looks incorrect. Maybe this?
```suggestion
            if (metric_spec.offset_window is not None or metric_spec.offset_to_grain is not None) and (
                metric_input_spec.offset_window is not None or metric_input_spec.offset_to_grain is not None
```

I'm not sure it matters if @courtneyholcomb will remove this and fix up immediately afterwards but we might as well fix it in case something has to get reverted or deployed quickly.

Reply:
Updated.